### DISCUSSION TITLE
You appear not to be near any campus location for the school you have selected.

### DISCUSSION THREAD

**Discussion by TuncVuralTuncay** [community/community#52829] [GitHub Education]
I submitted the GitHub Student package application with my school ID card but it was rejected. All the information is correct and my campus is near me but I see the error "your campus is not near you" in my gmail. Please help me. Thank you

**Comment by samus-aran** [comment]
Hi @Tuncvrjs 👋 

In order to qualify for the GitHub education program, we must be able to verify that you are either located close to your campus, or that you are a part of a verifiable distance learning program.

You will have the most success applying if you are able to do so from your school’s campus.

- When asked, please enable the application to discover your location from the browser pop-up. Since settings for every browser are different, if you are having difficulty enabling us to see your location you should seek assistance from your browser's support pages.
- If you are still having difficulties enabling your location check out [this discussion on assessing your hardware or operating system's settings.](https://github.com/orgs/community/discussions/42246)
- If you are using a VPN, please disable it when you reapply. 
- If you are not using a VPN, try reapplying when connected to the internet from a different network.

Can you confirm that your selected school's campus is near you?

**Reply by TuncVuralTuncay** [reply]
Hiiiii 👋. I don't use vpn, proxy or anything like that. My school is not on the list but I talked to my teacher and added my school to the list and tried again. But thank you very much anyway
